lib/translixir.ex

```diff
@@ -52,7 +55,7 @@ defmodule Translixir do
   def tx_log!(client, actions) when is_pid(client) do
     url = Client.endpoint(client, :tx_log)
     headers = Client.headers(client)
-    response = HTTPoison.post(url, "#{actions}", headers)
+    response = @http_client.post(url, "#{actions}", headers)
```

Comment:
Out of curiosity, Is there a reason to `actions` be wrapped by `string` in these cases?

Reply:
Edn format :(